governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Add wstETH-C as a new Vault Type - April 25, 2022
+summary: This poll proposes system parameters which could be used to initialize wstETH-C as a new vault type.
+discussion_link: https://forum.makerdao.com/t/introduction-risk-parameters-for-wsteth-c-vault/14669
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
```

Comment:
Can you add the "High Impact" category as well, please?

Reply:
Do we want to change the templates adding -[$Impact Level] ? I can work on that.